### DISCUSSION TITLE
PCB boards in the UK

### DISCUSSION THREAD

**Discussion by crazynightgriffin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#453] [General]
Don't suppose anyone has the PCB board and or other spare parts available in the UK?

Thanks in advance.

**Comment by bubble07** [comment]
I've got a spare version 1 board left over if that is any good to you?
I might have some of the other components as well, but will need to check at the weekend. 
Where abouts in the UK are you based?

**Reply by crazynightgriffin** [reply]
Thanks Bubble07, Still trying to get my head around the project the v1 covers the 6 cable version (although I do need to check my St Moritz square pump yet)

Based in Chester but we do get around a little as have a motorhome.

**Reply by bubble07** [reply]
Yeah, works with a 6 wire, mine is an S100102 6 wire square pump.
I'm not often in that direction, but happy to post, probably next week when I've got a bit more time

**Reply by crazynightgriffin** [reply]
> Yeah, works with a 6 wire, mine is an S100102 6 wire square pump.
> I'm not often in that direction, but happy to post, probably next week when I've got a bit more time

Cheers let me know if you find anything.

**Reply by urbanframe** [reply]
Hi @bubble07 

Sorry to hijack the thread. I've just got my S200102 Hydrojet working. I'm also going to build one for a friend that has a Hawaii (S100102). Can you remember how you wired up the 6 pin JST connectors?  Was it a straight ribbon cable from H1 and H2?  

With S100102 I had to cross a couple of wires. Thanks for any help you can give.
